apps/element-storybook/.storybook/main.ts

```diff
@@ -22,29 +24,39 @@ const config: StorybookConfig = {
         },
       },
     },
+    getAbsolutePath("@chromatic-com/storybook")
   ],
+
   framework: {
-    name: '@storybook/react-vite',
+    name: getAbsolutePath("@storybook/react-vite"),
     options: {
       builder: {
         viteConfigPath: '',
       },
     },
   },
+
   typescript: {
     reactDocgenTypescriptOptions: {
       propFilter: (prop) =>
         prop.parent
           ? !/node_modules\/(?!@mui)/.test(prop.parent.fileName) // filter out all node_modules except packages starting with "@mui"
           : true,
     },
+
+    reactDocgen: 'react-docgen-typescript'
   },
+
   staticDirs: ['../../../static'],
-  features: { storyStoreV7: false, legacyMdx1: true },
+  features: { storyStoreV7: false },
```

Comment:
I believe the legacy mdx was needed for the `<Source>` blocks in the bs4 migration files, unless the updates fixed the code block styling and we can go back to markdown code blocks.

Reply:
ugh... this was a bad merge. I will add that back.

Reply:
fixed